pkg/scheduler/metrics/metrics.go

```diff
@@ -104,6 +137,13 @@ func Register() {
 		prometheus.MustRegister(SchedulingAlgorithmPremptionEvaluationDuration)
 		prometheus.MustRegister(PreemptionVictims)
 		prometheus.MustRegister(PreemptionAttempts)
+
+		prometheus.MustRegister(AssumeVolumesLatency)
```

Comment:
Similarly here, can you add a RegisterMetrics call to the VolumeBinder interface, and put these inside there?  It would be good to keep the volume scheduler as separate from the main scheduling code as much as possible.

Reply:
Do  you mean we should extract these lines into a single function?

Reply:
Yes, I mean VolumeBinder should export a metrics registration interface that the scheduler can call.